app/models/user.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+
+from sqlmodel import Field, Index, SQLModel
+
+
+# Shared properties
+class UserBase(SQLModel):
+    name: str = Field()
+    email: str = Field(unique=True)
+    phone: str = Field(unique=True)
```

Comment:
Se permiten crear usuarios con:
- nombre vacío
- emails sin "@"
- telefono con valores no numéricos

Respecto al nombre, sumaría una condición de que al menos tenga N caracteres.
Para validar el teléfono e email, si te parece creamos 2 features de validación externos a este PR

```suggestion
class UserBase(SQLModel):
    name: str = Field(min_length=1, max_length=255)
```

Reply:
Creo USs [SCRUM-116](https://tpii-padel-pals.atlassian.net/browse/SCRUM-116) y [SCRUM-117](https://tpii-padel-pals.atlassian.net/browse/SCRUM-117) para la validación de la dirección de email y los telefonos.

Procedo a restringir la longitud de los nombres.